client/src/modules/authorization/components/SignInForm.jsx

```diff
@@ -175,6 +180,7 @@ class SignInForm extends PureComponent {
             formError={hasSignUpFailed}
             labelId="authorization.input.password.label"
             name="password"
+            noSuccessTheme
```

Comment:
what does it do?

Reply:
If the input is valid AuthInput displays this input in green. If there is no validator this field is always valid, so if the user enters something its appearance change to green. This is default behavior but in some special situations, this prop helps to prevent displaying a green color even if the input is valid.